test/unit/BPool.t.sol

```diff
@@ -1797,6 +1799,8 @@ contract BPool_Unit_SwapExactAmountIn is BasePoolTest {
     SwapExactAmountIn_FuzzScenario memory _fuzz
   ) public happyPath(_fuzz) {
     vm.assume(_tokenIn != VM_ADDRESS);
+    vm.assume(_tokenIn != tokenIn);
+    vm.assume(_tokenIn != tokenOut);
```

Comment:
is this `!= address(0)`?
can we implement `notFoundryAddress` here?

Reply:
We can implement it now, it will be implemented when merging #28 anyway

Reply:
#28 is for after `dev > main` PR